sites/blogs.txt

```diff
@@ -33,6 +36,7 @@ https://tech.trivago.com/
 https://www.intercom.com/blog/
 https://engineering.klarna.com/
 https://blog.algolia.com/
+https://www.thedroidsonroids.com/blog
```

Comment:
TBH, I don't think this should be added. It's a relatively small shop with what seems to be a heavily marketed blog. This section of the index is reserved for large, industry-defining companies with a strong, community-driven engineering culture.